flow-osgi/src/main/java/com/vaadin/flow/osgi/support/ServletContainerInitializerClasses.java

```diff
@@ -221,9 +220,11 @@ private void resetContextInitializers(ServletContext context) {
          * 
          * Lookup is set immediately in the context, so no need to initialize it
          */
-        initializerClasses.get().stream().filter(
-                clazz -> !clazz.equals(DevModeInitializer.class) && !clazz
-                        .equals(LookupServletContainerInitializer.class))
+        initializerClasses.get().stream().filter(clazz -> !clazz.getName()
+                .equals("com.vaadin.base.devserver.startup.DevModeInitializer")
```

Comment:
Since deb mode initializer now in the module which is not OSGi compatible this can be simply removed.

On the other hand : it may break the old Flow versions.

So let's go this way.